### PR TITLE
Fixed Grammar mistakes

### PR DESCRIPTION
**Pull request by synandi**
Fixed Grammar mistakes at lines 24, 26, 27 and 389

### CODE REVIEW

File: guides/serialization_and_saving.py [hunk 1]
```diff
@@ -386,7 +386,7 @@ def from_config(cls, config):
 
 In order to save/load a model with custom-defined layers, or a subclassed model,
 you should overwrite the `get_config` and optionally `from_config` methods.
-Additionally, you should use register the custom object so that Keras is aware of it.
```
Comment (fchollet) [inline guides/serialization_and_saving.py:389]: This one is indeed a typo -- please restrict the PR to this typo, and fix it in the `md` and `ipynb` files also. Thank you!

File: guides/serialization_and_saving.py [hunk 2]
```diff
@@ -21,10 +21,10 @@
 The Keras API makes it possible to save all of these pieces to disk at once,
 or to only selectively save some of them:
 
-- Saving everything into a single archive in the TensorFlow SavedModel format
+- Saves everything into a single archive in the TensorFlow SavedModel format
```
Comment (fchollet) [inline guides/serialization_and_saving.py:24]: Please keep "saving" in this list, it's what's intended.

Comment (synandi) [reply guides/serialization_and_saving.py]: I have Updated it with "Saving". Thank you!

**Review by fchollet — APPROVED** [review]
Thanks!

**Pull request merged**